shuup/simple_supplier/product_copy_signal_handler.py

```diff
@@ -9,12 +9,15 @@
 
 from shuup.admin.signals import product_copied
 from shuup.simple_supplier.models import StockCount
-
-
+        
 @receiver(product_copied, dispatch_uid="simple_supplier_product_copied")
-def handle_product_copy(sender, shop, copied, copy, **kwargs):
+def handle_product_copy(sender, shop, copied, copy, supplier=None, **kwargs):
     shop_product = copied.get_shop_instance(shop)
-    for supplier in shop_product.suppliers.all():
+    if supplier == None:
```

Comment:
do you really need to do this? `shop_product.suppliers.all()` will already have only a single supplier if there is a current supplier